process.c

```diff
@@ -4402,43 +4402,47 @@ rb_f_exit(int argc, const VALUE *argv)
 NORETURN(static VALUE f_exit(int c, const VALUE *a, VALUE _));
 /*
  *  call-seq:
- *     exit(status=true)
- *     Kernel::exit(status=true)
- *     Process::exit(status=true)
- *
- *  Initiates the termination of the Ruby script by raising the
- *  SystemExit exception. This exception may be caught. The
- *  optional parameter is used to return a status code to the invoking
- *  environment.
- *  +true+ and +FALSE+ of _status_ means success and failure
- *  respectively.  The interpretation of other integer values are
- *  system dependent.
- *
- *     begin
- *       exit
- *       puts "never get here"
- *     rescue SystemExit
- *       puts "rescued a SystemExit exception"
- *     end
- *     puts "after begin block"
+ *    exit(status = true)
+ *    Process::exit(status = true)
```

Comment:
I think we're standardizing on using a period rather than two colons for class methods right?

```suggestion
 *    Process.exit(status = true)
```

Reply:
Done.